### PR TITLE
#77 add Assign talks to event

### LINKED ISSUE #77: Assign talks to event
Event form - add possibility select (by checkboxes) assigned talks (only not yet assigned talks)

### PR DESCRIPTION
**Pull request by denys-medynskyi**
resolve #77

### CODE REVIEW

File: app/assets/stylesheets/admin/styles.scss [hunk 1]
```diff
@@ -66,6 +66,10 @@ form {
   width: 60px;
 }
 
+.block.checkbox label {
```
Comment (sveredyuk) [inline app/assets/stylesheets/admin/styles.scss]: .block is non-semantic div name... .form-actions or something more human readable..

Comment (sveredyuk) [reply app/assets/stylesheets/admin/styles.scss]: .checkboxes seems would be better

File: config/locales/en.yml [hunk 2]
```diff
@@ -83,6 +84,9 @@ en:
     success: Success
     failure: Failure
 
+  errors:
+    only_latin_letters: 'Only latin letters are allowed'
```
Comment (sveredyuk) [inline config/locales/en.yml]: i moved this messages to en-errors.yml. Pull fresh development and rebase

File: app/assets/stylesheets/admin/styles.scss [hunk 3]
```diff
@@ -66,6 +66,10 @@ form {
   width: 60px;
 }
 
+.checkboxes.checkbox label {
```
Comment (denys-medynskyi) [inline app/assets/stylesheets/admin/styles.scss:69]: @sveredyuk done

File: config/locales/en.yml [hunk 4]
```diff
@@ -24,6 +24,7 @@ en:
     is_true: 'yes'
     is_false: 'no'
     published: Published
+    description: Description
```
Comment (denys-medynskyi) [inline config/locales/en.yml:28]: @sveredyuk error removed

**Review by sveredyuk — APPROVED** [review]
@denysmedynskyi good job!

**Pull request merged**